src/OpenCvSharp/Internal/PInvoke/NativeMethods/core/NativeMethods_core.cs

```diff
@@ -167,6 +167,12 @@ static partial class NativeMethods
     public static extern ExceptionStatus core_normalize(IntPtr src, IntPtr dst, double alpha, double beta,
         int normType, int dtype, IntPtr mask);
 
+    [Pure, DllImport(DllExtern, CallingConvention = CallingConvention.Cdecl, ExactSpelling = true)]
+    public static extern ExceptionStatus core_reduceArgMax(IntPtr src, IntPtr dst, int axis, bool lastIndex);
```

Comment:
Maybe C++ bool types should be wrapped as UnmanagedType.U1.
https://learn.microsoft.com/en-us/visualstudio/code-quality/ca1414?view=vs-2022&tabs=csharp
```suggestion
    public static extern ExceptionStatus core_reduceArgMax(IntPtr src, IntPtr dst, int axis, [MarshalAs(UnmanagedType.U1)] bool lastIndex);
```

https://github.com/shimat/opencvsharp/blob/efefe1779ca7ce0c182e267dc02af9f494d8ca9a/src/OpenCvSharpExtern/core.h#L506

Reply:
Thanks for the hint!
I've updated and pushed accordingly.